### PR TITLE
Add coq-tactician to CI

### PR DESCRIPTION
**Pull request by LasseBlaauwbroek**
Just a draft to check if the CI works.

### CODE REVIEW

**Comment by SkySkimmer** [comment]
You also need to add it to the gitlab yml and to Makefile.ci

**Comment by LasseBlaauwbroek** [comment]
Now the plugin does show up in the pipeline, but there are no more minutes left so I guess I'll wait until there are (unless someone wants to do the cancel+restart dance).

I should also note that this plugin has a very large contact surface with the Coq API, so if you decide to merge this there will probably be a lot of PR's needed. At some point in the future, we could look at pushing some of my more generic code back into Coq to reduce this burden a bit (but for now I don't think the code is mature enough for that yet).

File: Makefile.ci [hunk 1]
```diff
@@ -59,7 +59,8 @@ CI_TARGETS= \
     ci-unimath \
     ci-unicoq \
     ci-verdi_raft \
-    ci-vst
+    ci-vst \
+    ci-tactician
```
Comment (SkySkimmer) [inline Makefile.ci:63]: list sould be alphabetically sorted

**Comment by ejgallego** [comment]
I think at the very least we should get the state rework we discussed done first, indeed the current hack is a bit too much IMHO as to ensure the plugins works.

I did a quick review and the rest doesn't seem to be too problematic.

**Comment by LasseBlaauwbroek** [comment]
> I think at the very least we should get the state rework we discussed done first

Agreed (although surprisingly in all the time between 8.10 and 8.14 this has never broken before :-P ). I will try to make some PR next week.

I forgot to add the executable bit to the script, so the build failed. Should be fixed now. But we can continue this when the hack is fixed.

**Comment by SkySkimmer** [comment]
Feel free to reopen or open a new PR when there is progress.

**Pull request closed without merging**